pontoon/base/views.py

```diff
@@ -290,12 +290,21 @@ def get_translations_from_other_locales(request):
     try:
         entity = request.GET['entity']
         locale = request.GET['locale']
+        plural_form = request.GET['plural_form']
```

Comment:
BTW, that is this change about?

Note that it breaks Translate.Current in its current form.

Reply:
This view doesn't take plural form into account. So if you're on a string that has a plural, you can only ever see the singular form from other locales. This change allows to see plural forms from other locales. I've also added a commit to fix my breaking the current Translate page.